dbms/src/Storages/StorageMergeTree.cpp

```diff
@@ -333,7 +339,8 @@ struct CurrentlyMergingPartsTagger
 
 void StorageMergeTree::mutate(const MutationCommands & commands, const Context &)
 {
-    MergeTreeMutationEntry entry(commands, full_path, data.insert_increment.get());
+    const auto full_paths = data.getFullPaths(); ///@TODO_IGR ASK What expected size of mutated part? what size should we reserve?
+    MergeTreeMutationEntry entry(commands, full_paths[0], data.insert_increment.get()); ///@TODO_IGR ASK PATH TO ENTRY
```

Comment:
As you decide

Reply:
What about expected size for such entry?